playground/src/app.js

```diff
@@ -24,7 +24,7 @@ function start() {
   registerScreens();
   Navigation.events().onAppLaunched(() => {
     Navigation.setDefaultOptions({
-      _animations: {
+      animations: {
```

Comment:
Lets avoid using custom animations in the playground app for now.
If you really want to use them, let's make the animation look decent :)